### ISSUE TITLE
Worker docs has 2 different endpoint regex for same endpoint

### ISSUE THREAD

**Issue by olmari** [matrix-org/synapse#12229]
At https://github.com/matrix-org/synapse/blob/develop/docs/workers.md (master is same in this regard at the moment) there exist 2 different regex endpoints for some endpoints. Mostly those that is also mentioned in stream writers has more regex matches than the otherwise same endpoint given earlier in generic worker list..

For example under generic worker listing:
```
# Device requests
^/_matrix/client/(r0|v3|unstable)/sendToDevice/
```
and then later in stream writer `to_device` listing:
```
^/_matrix/client/(api/v1|r0|v3|unstable)/sendToDevice/
```

**Comment by clokep** [comment]
> ^/_matrix/client/(r0|v3|unstable)/sendToDevice/

Seems to be the correct one, as the endpoint is not registered for v1:

https://github.com/matrix-org/synapse/blob/d1f1b46c2cfe612790aebdd54765953951c94e45/synapse/rest/client/sendtodevice.py#L35-L37

Would you mind making a PR with the fix? If not, I'll try to do it soon.

**Comment by olmari** [comment]
I could, if I can know are rest of the doubly mentioned endpoints without `api/v1` correct for all of them :)

**Comment by clokep** [comment]
I ended up making a PR with the fixes: #12243.
